config/config.hocon.sample

```diff
@@ -37,6 +37,9 @@
       # Shredder output compression, GZIP or NONE
       "compression": "GZIP"
     }
+    # ARN of SNS topic used by Shredder.
```

Comment:
In enrich when a config field is optional we mark it straightaway =>
```
# Optional. ARN of SNS topic used by Shredder.
```

Reply:
https://github.com/snowplow/snowplow-rdb-loader/pull/594#discussion_r742827852